src/huggingface_hub/inference/_client.py

```diff
@@ -2444,17 +2445,35 @@ def zero_shot_classification(
             ZeroShotClassificationOutputElement(label='robots', score=0.00030448526376858354),
         ]
         ```
+
+        Example for multi_label=True and custom hypothesis_template:
+        ```py
+        >>> from huggingface_hub import InferenceClient
+        >>> client = InferenceClient()
+        >>> output = client.zero_shot_classification(
```

Comment:
no need to assign to a tmp variable. Printing it directly is fine for a code example. The `>>>` syntax suggests the code snippet is run in a terminal so outputs are always printed (except if None)
```suggestion
        >>> client.zero_shot_classification(
```